### ISSUE TITLE
ES6 / JS modules example

### ISSUE THREAD

**Issue by btopro** [inorganik/countUp.js#215]
Is there a way to implement this in ES6 browser processed modules? Trying:
```
import * as CountUp from 'countup.js'
```
and getting:
```
Uncaught ReferenceError: exports is not defined at countUp.min.js:11
```

**Comment by inorganik** [comment]
It should work as 

```js
import { CountUp } from 'countup.js';
```
Let me know

**Comment by btopro** [comment]
```js
Uncaught SyntaxError: The requested module '../../countup.js/dist/countUp.min.js' does not provide an export named 'CountUp'
```

**Comment by inorganik** [comment]
I looked into it, and have not figured it out yet. I asked on SO: https://stackoverflow.com/questions/55167994/is-it-possible-to-es6-import-a-commonjs-module?noredirect=1#comment97075315_55167994

**Comment by btopro** [comment]
Thank you. for context, this would make it easy to integrate with webcomponents and ship natively on the front-end without really needing to do much in the way of tooling.

Until this is worked out for the time being I've included the base of the code directly though I'd much rather want to reference and pull it in for sustainability purposes (this component was mostly to demonstrate what's gained / potential of this approach) -- https://www.webcomponents.org/element/@lrnwebcomponents/count-up/demo/demo/index.html

Seeing that you've worked on React / Angular wrappers I figured I'd make the WC one to get it up and running. It's built in Polymer 3.

**Comment by inorganik** [comment]
That's awesome, thanks for working on it! Since I re-wrote CountUp in typescript, the modules have been a HUGE headache. ([Issue thread](https://github.com/inorganik/countUp.js/issues/210)) Before 2.X, I used `gulp-umd-wrapper` which produced a UMD module and globally scoped `CountUp` which is very compatible for everyone... Typescript wrapped UMD module does not make `CountUp` global, which makes bad compatibility, especially with Webpack.

So I released again with an `AMD` wrapper, it seemed to be compatible with both requirejs and Webpack. But people still had issues. So I released **again** with commonjs module wrapping. I found it works well with Browserify and Webpack. 

I did some tests and was able to use es6 export successfully with vanilla js and webpack. Perhaps all issue another release...
